tools/lxdclient/utils.go

```diff
@@ -65,6 +66,8 @@ func IsRunningLocally() (bool, error) {
 	return running, nil
 }
 
+const errIPV6NotSupported = `cannot listen on https socket: listen tcp \[::\]:\d*: socket: address family not supported by protocol`
```

Comment:
I don't have a particularly better regex to use. This feels a little brittle (version of Go, version of LXD API, platform Trusty vs Xenial, etc.)
Can we get some sort of Functional test so that we have an idea when this string is no longer valid and can update it?